cleancode/src/main/java/com/epam/cleancode/comments/MortgageInstallmentCalculator.java

```diff
@@ -2,37 +2,24 @@
 
 public class MortgageInstallmentCalculator {
 
-    /**
-     *
-     * @param p principal amount
-     * @param t term of mortgage in years
-     * @param r rate of interest
-     * @return monthly payment amount
-     */
-    public static double calculateMonthlyPayment(
-            int p, int t, double r) {
+    public static double calculateMonthlyPayment(int principalAmount, int termMortgageInYears, double rateOfInterest) {
+        double termMortgageInMonths;
+        double monthlyRateOfInterest;
+        double monthlyPayment;
 
-        //cannot have negative loanAmount, term duration and rate of interest
-        if (p < 0 || t <= 0 || r < 0) {
+        if (principalAmount < 0 || termMortgageInYears <= 0 || rateOfInterest < 0) {
             throw new InvalidInputException("Negative values are not allowed");
         }
 
-        // Convert interest rate into a decimal - eg. 6.5% = 0.065
-        r /= 100.0;
+        rateOfInterest /= 100.0;
+        termMortgageInMonths = termMortgageInYears * 12;
```

Comment:
12 is magic number

Reply:
`final int YEARINMONTHS = 12;
 ...
 rateOfInterest /= 100.0;
 termMortgageInMonths = termMortgageInYears * YEARINMONTHS;`
fix https://github.com/TashaB1/EpamTraining2019/commit/e2848c1f5ec0ec7697069142242c0daa4c532733